ui/apps/platform/src/Containers/Vulnerabilities/NodeCves/Overview/CVEsTable.tsx

```diff
@@ -50,7 +51,7 @@ export const sortFields = [
     // FIRST_DISCOVERED_SORT_FIELD,
 ];
 
-export const defaultSortOption = { field: NODE_TOP_CVSS_SORT_FIELD, direction: 'desc' } as const;
+export const defaultSortOption = { field: CVE_SORT_FIELD, direction: 'desc' } as const;
```

Comment:
I changed the default sort because currently the other sort options on the Node CVE table throw errors.

Top CVSS:
![image](https://github.com/stackrox/stackrox/assets/1292638/a8d47605-b177-4c07-bcfb-60e97b3cb329)

Affected nodes:
![image](https://github.com/stackrox/stackrox/assets/1292638/f96efab0-e29a-4ee2-8f75-47e40de3ccf4)

@charmik-redhat FYI - let me know if I should create Jira tickets for these or if there is more info you need.